phantomjs-env/src/test/scala/org/scalajs/jsenv/phantomjs/PhantomJSSuite.scala

```diff
@@ -0,0 +1,16 @@
+package org.scalajs.jsenv.phantomjs
+
+import org.scalajs.jsenv.test._
+
+import org.junit.runner.RunWith
+
+@RunWith(classOf[JSEnvSuiteRunner])
+class PhantomJSSuite extends JSEnvSuite(PhantomJSSuite.Config)
+
+object PhantomJSSuite {
+  val Config = {
+    JSEnvSuiteConfig(new PhantomJSEnv)
+      .withTerminateVMJSCode("__ScalaJSEnv.exitFunction(0)")
+      .withSupportsTimeout(false)
```

Comment:
Why this? I assume the tests are too restrictive? I also have trouble with the timeout tests on Selenium. So probably we need to improve the test suite there.

Reply:
I'm not sure, I haven't investigated more than the fact that some tests failed, since the status before that PR was also not to execute the Timeout tests:
https://github.com/scala-js/scala-js-env-phantomjs/blob/6a1e55846b84b20dc51de7b9693db78feea1b50c/phantomjs-env/src/test/scala/org/scalajs/jsenv/phantomjs/PhantomJSTest.scala#L7
only extends `ComTests`, not `TimeoutComTests`.

Reply:
OK, fair enough.

Reply:
Maybe create an issue and link it here to enable this, once scala-js/scala-js#3420 is fixed?

Reply:
Done (#20).